account_billing/report/report_billing.xml

```diff
@@ -0,0 +1,74 @@
+<odoo>
+
+    <template id="report_billing_document">
+        <t t-call="web.external_layout">
+            <t t-set="address">
+                <address t-field="o.partner_id" t-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": True}' />
+                <div t-if="o.partner_id.vat" class="mt16"><t t-esc="o.company_id.country_id.vat_label or 'Tax ID'"/>: <span t-field="o.partner_id.vat"/></div>
+            </t>
+            <div class="page">
+                <h2>
+                    <span t-if="o.state == 'billed'">Billing</span>
+                    <span t-if="o.state == 'draft'">Draft Billing</span>
+                    <span t-if="o.state == 'cancel'">Cancelled Billing</span>
+                    <span t-field="o.name"/>
+                </h2>
+
+                <div id="informations" class="row mt32 mb32">
+                    <div class="col-auto mw-100 mb-2" t-if="o.date" name="billing_date">
+                        <strong>Billing Date:</strong>
+                        <p class="m-0" t-field="o.date"/>
+                    </div>
+                    <div class="col-auto mw-100 mb-2" t-if="o.due_date" name="due_date">
+                        <strong>Due Date:</strong>
+                        <p class="m-0" t-field="o.due_date"/>
+                    </div>
```

Comment:
`o.due_date` now does not exist. It should be `o.threshold_date`.

The label should also be changed to "Threshold Date". Wasn't sure if you'd like to adjust the label according to the type of the threshold date.

Reply:
Thank for review